docs/pages/admin-guides/access-controls/access-requests/resource-requests.mdx

```diff
@@ -307,7 +307,7 @@ below, edit one of the user's roles so the `search_as_roles` field includes the
 role you have created.
 
 For full details on how to use Teleport roles to configure RBAC, see the
-[Teleport Access Controls
+[Access Controls
```

Comment:
Same question re: a Vale config. Are we avoiding "Teleport Access Controls" everywhere?